client/src/annoMatrix/views.ts

```diff
@@ -135,45 +177,59 @@ class AnnoMatrixMapView extends AnnoMatrixView {
 }
 
 export class AnnoMatrixClipView extends AnnoMatrixMapView {
+  clipRange: [number, number];
+
+  isClipped: boolean;
+
   /*
-	A view which is a clipped transformation of its parent
-	*/
-  // @ts-expect-error ts-migrate(7006) FIXME: Parameter 'viewOf' implicitly has an 'any' type.
-  // eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types --- FIXME: disabled temporarily on migrate to TS.
-  constructor(viewOf, qmin, qmax) {
-    // @ts-expect-error ts-migrate(7006) FIXME: Parameter 'field' implicitly has an 'any' type.
-    super(viewOf, (field, colLabel, colSchema, colData, df) =>
-      _clipAnnoMatrix(field, colLabel, colSchema, colData, df, qmin, qmax)
+    A view which is a clipped transformation of its parent
+    */
+  constructor(viewOf: AnnoMatrix, qmin: number, qmax: number) {
+    super(
+      viewOf,
+      (
+        field: Field,
+        colLabel: string,
+        colSchema: ArraySchema,
+        colData: string[],
```

Comment:
this isn't a string[].   It should be AnyArray, as it is the underlying array data from a dataframe column.

Longer term, we need to flow some typing through from the caller (ie, _doLoad()).   If you leave a big TODO comment in the code, I'll revisit this once we merge this PR and the Dataframe PR.

Reply:
In the new Dataframe PR, it is `DataframeValueArray`

Reply:
Updated [views.ts:constructor](https://github.com/chanzuckerberg/cellxgene/blob/mim/annoMatrix/client/src/annoMatrix/views.ts#L180).